packages/themes/tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "baseUrl": ".",
     "target": "ES2022" /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
     "module": "ESNext" /* Specify what module code is generated. */,
-    "moduleResolution": "Node",
+    "moduleResolution": "Bundler",
```

Comment:
hey great work! do you think you can do one better and just extend the base tsconfig?

```json
{
  "extends": "../../tsconfig.json"
}
```